controller/controller_server/messages.py

```diff
@@ -20,6 +20,6 @@
 volume_id_wrong_format_message = 'volume id has wrong format'
 readonly_not_supported_message = 'readonly parameter is not supported'
 node_id_wrong_format_message = 'node id has wrong format'
-volume_src_snapshot_id_is_missing = 'volume source snapshot id is missing'
+volume_src_object_id_is_missing = 'volume source {0} id is missing'
```

Comment:
this is a good opportunity to rename `src` to `source`, and consider removing `object`

Reply:
done